pyomo/contrib/appsi/solvers/tests/test_persistent_solvers.py

```diff
@@ -819,28 +820,47 @@ def test_time_limit(self, name: str, opt_class: Type[PersistentSolver]):
         opt: PersistentSolver = opt_class()
         if not opt.available():
             raise unittest.SkipTest
-        if type(opt) is Cbc:
-            raise unittest.SkipTest
         from sys import platform
         if platform == 'win32':
             raise unittest.SkipTest
+
+        if opt.available() == opt.Availability.LimitedLicense:
+            N = 30
+        else:
+            N = 100
```

Comment:
Is there a reason to make the model bigger when a license is present?  If the test reliably times out with the smaller model, it should do the same with the larger one, right?

Reply:
It seems to me that different solvers check the time limit at different times. I think you are still correct though. I think this should work with `N=30` for all solvers.